azure-pipelines.yml

```diff
@@ -1,23 +1,3 @@
-# This is a simple wrapper for eng/pipeline.yml to get around the limitation of
-# user-defined variables not being available in yaml template expressions.
-
-# Parameters ARE available in template expressions, and parameters can have default values
-variables:
-# clean the local repo on the build agents
-- name: Build.Repository.Clean
-  value: true
-- name: _DotNetArtifactsCategory
-  value: WINDOWSDESKTOP
-- name: _DotNetValidationArtifactsCategory
-  value: WINDOWSDESKTOP
-- ${{ if or(startswith(variables['Build.SourceBranch'], 'refs/heads/release/'), startswith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), eq(variables['Build.Reason'], 'Manual')) }}:
-  - name: PostBuildSign
-    value: false
-- ${{ else }}:
-  - name: PostBuildSign
-    value: true
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - group: DotNet-Wpf-SDLValidation-Params
 trigger:
```

Comment:
Probably a comment is needed here to indicate that the official build does no actual building of real shipping assets.  We rely on VMR to do that.